en/middleware/micrortps.md

```diff
@@ -34,6 +33,25 @@ The *Client* application is also compiled and built into the firmware as part of
 > **Tip** The bridge code can also be [manually generated](micrortps_manual_code_generation.md). Most users will not need to do so, but the linked topic provides a more detailed overview of the build process and can be useful for troubleshooting.
 
 
+## Supported uORB messages
```

Comment:
I don't think we should move this section. A normal developer who wants to use RTPS does not need this information to build and use the bridge - because the standard set is already defined. This information would be useful as "additional detail", which is why it was in the other document